### DISCUSSION TITLE
Netbox Device (router) connection to ipam v4/v6

### DISCUSSION THREAD

**Discussion by tcsmithh** [netbox-community/netbox#16578] [Help Wanted!]
i have done some simple searches to determine if there is way to dynamically connect a device to ipam (v4 and v6). what i would like to do is create a device and assign its address and have ipam enter the addresses from the device when it is created. 
thanks for the help :)

**Comment by hackcl** [comment]
From my unterstanding you cannot just add an IP Address to a device. You need to add an interface to the device and then add an IP to the interface. The IP is then binded to the interface but not directly to the device.
